include/sparrow/layout/variable_size_binary_array.hpp

```diff
@@ -184,47 +194,50 @@ namespace sparrow
     public:
 
         using self_type = variable_size_binary_array<T, CR, OT>;
-        using base_type = array_bitmap_base<self_type>;
+        using base_type = array_bitmap_base_impl<self_type, false>;
```

Comment:
Since `array_bitmap_base<T>` is an alias of `array_bitmap_base_impl<T, false>`, I think this change can be reverted.